src/data/data.hpp

```diff
@@ -59,7 +59,7 @@ typedef float nav_t;
 typedef Vector<nav_t, 3> NavigationVector;
 struct Navigation : public Module {
   static constexpr float run_length = 1250;  // m
-  static constexpr float braking_buffer = 20; // m
+  static constexpr float braking_buffer = 20;  // m
```

Comment:
You might want to add to the PR message that you also fixed some linter issues in data and state machine with white space : )

Reply:
Also, try to keep the predefined formatting for the PR message when opening it. Just maintains consistency and gives a nice little description and info about the pull request